mods/action.js

```diff
@@ -181,6 +181,13 @@ function buildEditResponse(res) {
 }
 
 ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
+    if (/\.test\.local$/.test(req.params.domain)) {
```

Comment:
@d00rman I don't like these too, but how do you imaging it? We would then need 2 different sys specs for different domains, which would be extremely verbose.